.gitignore

```diff
@@ -96,4 +96,9 @@ sdk/storage/azure-storage-file-datalake/tests/settings_real.py
 sdk/cosmos/azure-cosmos/test/test_config.py
 
 # temp path to to run regression test
-.tmp_code_path/
\ No newline at end of file
+.tmp_code_path/
+
+# api view
```

Comment:
this may not be harmful but I'd be hesitant to adjust the gitignore for something slightly more special cased.

Reply:
🤔 I don't remember making this change but I'll unmake it